refinery/core/models.py

```diff
@@ -1362,19 +1390,19 @@ def attach_outputs_downloads(self):
             return
 
         for analysis_result in self.results.all():
-            item = FileStoreItem.objects.get(
-                uuid=analysis_result.file_store_uuid
-            )
-            if item:
+            try:
```

Comment:
@jkmarx Given that we know (off-the-code) that there may as well be a foreign key relationship to file_store_uuid, i think that all these try-catches are likely never going to be used since an AnalysisResult essentially can't exist without a FileStoreItem